.github/workflows/deno.yml

```
@@ -0,0 +1,88 @@
+      - ".github/workflows/test.yml"
+  workflow_dispatch:
+    inputs:
+      denops_branch:
+        description: 'Denops branch to test'
+        required: false
+        default: 'main'
+
+# Use 'nushell' as it's the only sane shell
+defaults:
+  run:
+    shell: nu {0} 
```

yamllint

[error] trailing spaces

(trailing-spaces)